python/lsst/dax/apdb/scripts/create_cassandra.py

```diff
@@ -56,6 +57,9 @@ def create_cassandra(
     config = ApdbCassandra.init_database(ra_dec_columns=ra_dec_list, table_options=options, **kwargs)
     config.save(output_config)
 
+    apdb = ApdbCassandra.from_config(config)
+    apdb.metadata.set("instrument", instrument)
+
```

Comment:
This code should go in `init_database`; otherwise calls that _don't_ go through the CLI will miss it. This function should be command line parsing only.

It's possible that the right place for this is actually [`_makeSchema`](https://github.com/lsst/dax_apdb/blob/main/python/lsst/dax/apdb/cassandra/apdbCassandra.py#L658-L665), since that's what sets the existing metadata. But I think that's an @andy-slac question.